about.html

```diff
@@ -12,52 +12,69 @@
 </head>
 
 <body>
+  <header>
+    <section class="header-container">
+        <div class="header-top">
+            <img src="img/lambda-black.png" class="header-image" alt="Lambda title">
+        </div>
+        <div class="header-top-right">   
+            <nav>
+                <a class="home container-block" href="index.html">Home</a>
+                <a class="about container-block" href="about.html">About</a>
+                <a class="products container-block" href="#">Products</a>
+                <a class="blog container-block" href="#">Blog</a>
+                <a class="contact container-block" href="#">Contact</a>
+            </nav>
+        </div>
+        <section class="bottom-container-image">
+            <img src="img/jumbo-about.png" class="bottom-header-image" alt="Lambda bottom header">
+</header>
     <div class="container about-page">
      
-    About
+    <h2>About</h2>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
 
     <img src="img/about-plan.png" alt="strategy">
```

Comment:
Your doing a good job with this.  If you wrap each of these sections; strategy, how we work, places we work and collaboration in their own divs (or semantic elements) they will be able to be targeted in CSS which will help you to apply flexbox styles to them to get the layout like the design file.